start_local_server.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -e
+export DOCKER_SCAN_SUGGEST=false
+
+docker compose run --rm app composer install --ignore-platform-req=ext-gd --ignore-platform-req=ext-imagick
```

Comment:
why do u need to ignore the listed ext-*  u need to install it on docker please check
https://github.com/OpenStackweb/summit-api/pull/265/files#r1631148173